nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/test/java/org/apache/nifi/processors/azure/storage/ITDeleteAzureDataLakeStorage.java

```diff
@@ -368,10 +438,19 @@ private boolean fileExists(String directory, String filename) {
         return fileClient.exists();
     }
 
-    private void setRunnerProperties(String fileSystem, String directory, String filename) {
+    private boolean directoryExists(String directory) {
+        DataLakeDirectoryClient directoryClient = fileSystemClient.getDirectoryClient(directory);
+
+        return directoryClient.exists();
+    }
+
+    private void setRunnerProperties(String fileSystem, String directory, String filename, boolean isFile) {
+        runner.setProperty(DeleteAzureDataLakeStorage.FILESYSTEM_OBJECT_TYPE, isFile ? "file" : "folder");
```

Comment:
"folder" does not work because the AllowableValue has been renamed to "directory".
Using `AllowableValue.getValue()` would eliminate these renaming issues.